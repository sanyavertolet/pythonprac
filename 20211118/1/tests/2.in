@objcount
class Gromozeka:
    def __init__(self, arg = 'BZZZ'):
        print('ME SAY', arg)
        self.phrase = arg
    def __del__(self):
        print('ME NO SAY', self.phrase, 'ANYMORE :c')

a = Gromozeka()
b = Gromozeka('YEEEEEEY')
c = Gromozeka('RED GOES FASTA')
print(Gromozeka.counter)
a = 0
b = 0
print(Gromozeka.counter)
c = 0
print(Gromozeka.counter)

