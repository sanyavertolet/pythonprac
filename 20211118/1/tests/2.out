ME SAY BZZZ
ME SAY YEEEEEEY
ME SAY RED GOES FASTA
3
ME NO SAY BZZZ ANYMORE :c
ME NO SAY YEEEEEEY ANYMORE :c
1
ME NO SAY RED GOES FASTA ANYMORE :c
0
